ios/help/about/whatsnew.md

```diff
@@ -4,9 +4,4 @@ title: What's New
 
 Here are some of the new features we have added to Keyman for iPhone and iPad 16.0:
```

Comment:
```suggestion
Here are some of the new features and major bug fixes we have added to Keyman for iPhone and iPad 16.0:
```

A number of these pages list things that are very clearly _not_ features, but significant bug fixes.